<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWriteMSUIClass</class>
 <widget class="QMainWindow" name="CWriteMSUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CWriteMSUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>218</width>
         <height>280</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="pSenderLabel">
          <property name="text">
           <string>发件人：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="pSenderLineEdit">
          <property name="text">
           <string>13801234567</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="pReceiverLabel">
          <property name="text">
           <string>收件人：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="pReceiverLineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="pThemeLabel">
          <property name="text">
           <string>主   题：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="pThemeLineEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTextBrowser" name="pmessagetextBrowser">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonBack">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPage">
             <property name="text">
              <string>1/1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonNext">
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="pactionExit"/>
    <addaction name="separator"/>
    <addaction name="pactionSend"/>
    <addaction name="pactionSave"/>
    <addaction name="separator"/>
    <addaction name="pactionBack"/>
    <addaction name="pactionNext"/>
    <addaction name="separator"/>
    <addaction name="pactionInsertImage"/>
    <addaction name="pactionAddPage"/>
    <addaction name="pactionDelPage"/>
    <addaction name="pactionAddUser"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="pactionExit">
   <property name="text">
    <string>取消|退出</string>
   </property>
  </action>
  <action name="pactionSend">
   <property name="text">
    <string>发送</string>
   </property>
  </action>
  <action name="pactionNext">
   <property name="text">
    <string>下一页</string>
   </property>
  </action>
  <action name="pactionInsertImage">
   <property name="text">
    <string>插入图片</string>
   </property>
  </action>
  <action name="pactionAddPage">
   <property name="text">
    <string>添加一页</string>
   </property>
  </action>
  <action name="pactionDelPage">
   <property name="text">
    <string>删除一页</string>
   </property>
  </action>
  <action name="pactionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="pactionBack">
   <property name="text">
    <string>上一页</string>
   </property>
  </action>
  <action name="pactionAddUser">
   <property name="text">
    <string>收件人</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pactionExit</sender>
   <signal>triggered()</signal>
   <receiver>CWriteMSUIClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
